example-code/snippets/src/backend/SerializingXmlSchemas.ts

```diff
@@ -14,9 +14,9 @@ export class SchemaXmlSerializer {
   public async serializeSchemaToXmlSample() {
```

Comment:
@ColinKerr Should we also reference SchemaXml.writeFile function from ecschema-locaters as an alternative to this?

Reply:
This is the documentation for if you do not want to use those helpers.  See serializing-xml-schemas.md for the docs that reference this

Reply:
Got it.  Thanks.